src/pages/Articles.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from "react";
+import { getAllArticles } from "../api/articles_api";
+import TextHeading from "../components/misc/TextHeading";
+import ArticleCarouselItem from "../components/articles/ArticleCarouselItem";
+import AliceCarousel from "react-alice-carousel";
+import {
+  NextButton,
+  PrevioustButton,
+} from "../components/articles/CarouselButtons";
+import "react-alice-carousel/lib/alice-carousel.css";
+import "./Articles.scss";
+import parse from "rss-to-json";
+
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+
+  const parseItem = (item) => {
+    const parser = new DOMParser();
+    document = parser.parseFromString(item.content, "text/html");
+
+    console.log(document);
+
+    return item;
+  };
+
+  useEffect(() => {
+    (async () => {
+      const articles = await getAllArticles();
+      setArticles(articles);
+    })();
+  }, []);
+
+  return (
+    <div>
+      <TextHeading text="For You From Chainwhiz" />
+      <AliceCarousel
+        autoWidth
+        mouseTracking
+        touchTracking
+        disableDotsControls={true}
+        renderNextButton={(_) => <NextButton />}
+        renderPrevButton={(_) => (true ? null : <PrevioustButton />)}
```

Comment:
Can you omit the **true condition** and keep the PreviousButton? Also, fix the CSS for the same

Reply:
For both the next and previous buttons, make the cursor as pointer.

Reply:
Hello @rajashree23, what should be the position of the previous button? Left corner of the screen, right?

Reply:
Yes @prathamesh-mutkure .
Please refer this link [https://www.figma.com/file/qIczRAb17s1dVaF3NOSpOO/Chainwhiz-Articles](https://www.figma.com/file/qIczRAb17s1dVaF3NOSpOO/Chainwhiz-Articles)
![image](https://user-images.githubusercontent.com/33730790/160231351-7dd9d261-ee71-4932-bbec-4459b4691a59.png)